src/index.js

```diff
@@ -10,21 +10,28 @@
  *                               (defaults to true)
  * @property {String} dataLayerInstanceName The name of the data ayer instance in the global scope
  *                                          (defaults to "adobeDataLayer")
+ * @property {Boolean} includeDataLayerState Whether to include the datalayer state on every
+ *                                           event that is sent by alloy (defaults to true)
  * @property {String[]} launchUrls A list of launch container URLs to load (defults to empty list)
  * @property {Boolean} personalization Indicates whether Adobe Target should be enabled
  *                                     (defaults to true)
  * @property {Number} personalizationTimeout Indicates the amount of time to wait before bailing
  *                                           out on the personalization and continue rendering the
  *                                           page (defaults to 1s)
+ * @property {Boolean} performanceOptimized Whether to use the agressive performance optimized
+ *                                          instrumentation, or the more traditional alloy approach
+ *                                          (defaults to true)
  */
 export const DEFAULT_CONFIG = {
   analytics: true,
   alloyInstanceName: 'alloy',
   dataLayer: true,
   dataLayerInstanceName: 'adobeDataLayer',
+  includeDataLayerState: true,
```

Comment:
Setting new parameter to toggle whether the data layer state should be included on all alloy events or not